src/TimeGrid.jsx

```diff
@@ -77,9 +80,24 @@ export default class TimeGrid extends Component {
       this.measureGutter()
     }
 
+    if (this._scrollRatio) {
+      const { content } = this.refs;
+      content.scrollTop = content.scrollHeight * this._scrollRatio;
+      // Only do this once
+      this._scrollRatio = null;
```

Comment:
I calculate the new position and set it on [`componentWillMount`](https://github.com/intljusticemission/react-big-calendar/pull/122/files#diff-fe4a2572cc6abfc6a7df5455b234527eR67) and [`componentWillReceiveProps`](https://github.com/intljusticemission/react-big-calendar/pull/122/files#diff-fe4a2572cc6abfc6a7df5455b234527eR97). Then later, on [`componentWillUpdate`](https://github.com/intljusticemission/react-big-calendar/pull/122/files#diff-fe4a2572cc6abfc6a7df5455b234527eR83) (I can't do it before because changes haven't been flushed to the DOM) apply the actual scroll. What do you mean by both? `componentWillUpdate` gets called on all cases. Also I applied this to the `selectable` example so you can test it working youself.